spec/forms/prior_authority/steps/service_cost_form_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe PriorAuthority::Steps::QuoteCostForm do
+  subject(:form) { described_class.new(attrs) }
+
+  let(:application) { build(:prior_authority_application, :with_primary_quote) }
+
+  let(:attrs) do
+    { user_chosen_cost_type: 'by_item',
+      items: 2,
+      cost_per_item: 3,
+      period: nil,
+      cost_per_hour: nil,
+      application: application,
+      record: application.primary_quote }
+  end
+
+  describe '#cost_type' do
+    it 'returns cost_type' do
```

Comment:
Nit: This is not a good test. First of all, the text says "#cost_type returns cost_type", which is meaningless. Second, the code appears to demonstrate that #cost_type returns the input `user_chosen_cost_type`. But this only happens in a very specific set of circumstances, namely if the service_type is associated with a variable cost type, which _happens_ to be the case with the test fixture you're using but isn't made at all explicit in the test. So as it stands this is pretty misleading.

Reply:
yeah, binned